pkg/datastore/database.go

```diff
@@ -61,6 +61,7 @@ Postgres-backed implementation of DataStore interface. By default, uses Metadata
 */
 type relationalDb struct {
 	sync.RWMutex
+	dbName      string
```

Comment:
I could always print out DB name by running a SQL command that returns the DB name, but it would be faster to just store DB name as a field of this struct

Reply:
lgtm